examples/render-helm-chart-kustomize-inline-values/kustomization-dir/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+generators:
```

Comment:
Do we need to keep `kustomization.yaml` in a separate directory? Can we keep in flat?